client-extensions/liferay-object-collection-filter-extension/src/common/components/search/FilterForm.js

```diff
@@ -0,0 +1,22 @@
+import React, {useEffect, useState} from 'react';
+import FilterInput from "./FilterInput";
+import {getFilterValue} from "../../util/FilterUtil";
+
+const FilterForm = ({ selectedFields, onUpdateFilter, filters}) => {
+    return (
+        <div className="filter-list">
+            {selectedFields.map((field) => (
+                <div className="d-inline-block mr-2">
```

Comment:
should have key for each item.
[FYI](https://dev.to/danielbellmas/react-key-not-only-in-map-2d7e#:~:text=Setting%20key%20will%20inform%20React,are%20added%2C%20or%20are%20removed.)